src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,16 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        Lottery lottery = new Lottery();
+        Ball[] balls = new Ball[3];
+
+        for (int i = 0; i < 3; i++) {
+            balls[i] = lottery.getRandomBall();
+        }
+
+        for (Ball ball : balls) {
+            System.out.println(ball.toString());
+        }
+
```

Comment:
Adding an empty line after each block is not very cool. Let's remove the excess

Reply:
Funny, I thought that if mvn does not scold me for it, then I did well, but I was wrong, I will try not to do so